### PR TITLE
Add system information #2

### PR DESCRIPTION
**Pull request by Meliox**
Implements #45 

See review in #46

### CODE REVIEW

File: pve-mod-gui-sensors.sh [hunk 1]
```diff
@@ -633,13 +659,35 @@ Ext.define('PVE.mod.TempHelper', {\n\
...
 	},
 		}" "$pvemanagerlibjs"
 		fi
 
+		if [[ $enableSystemInfo == "true" ]]; then
+			sed -i "/^Ext.define('PVE.node.StatusView',/ {
+				:a;
+				/title:/!{N;ba;}
+				:b;
+				/'separatorBox.*},/!{N;bb;}
```
Comment (Meliox) [inline pve-mod-gui-sensors.sh:675]: Currently dependent on separator box. Should be before the block, so sed should be updated:





	{
	    colspan: 2,
	    title: gettext('Kernel Version'),
	    printBar: false,
	    // TODO: remove with next major and only use newish current-kernel textfield
	    multiField: true,
	    //textField: 'current-kernel',
	    renderer: ({ data }) => {
		if (!data['current-kernel']) {
		    return data.kversion;
		}
		let kernel = data['current-kernel'];
		let buildDate = kernel.version.match(/\((.+)\)\s*$/)?.[1] ?? 'unknown';
		return `${kernel.sysname} ${kernel.release} (${buildDate})`;
	    },
	    value: '',
	},

**Comment by Meliox** [comment]
@eremem: I resolved my comment. Everything should be working. Let me know if you see something I forgot before merging

**Comment by eremem** [comment]
@Meliox I'll do the review of this and the other 2 PRs ASAP. I'll try to have a look at them within the next few days.

**Review by eremem — APPROVED** [review]
This looks really cool! 😄

**Comment by Meliox** [comment]
Thanks! Maybe we should consider renaming the script to pve-mod-system-information...

**Comment by eremem** [comment]
> Thanks! Maybe we should consider renaming the script to pve-mod-system-information...
I'd stick to the "place" we modify, so maybe pve-mod-node-status, pve-mod-node-info or pve-mod-node-status-view? 😃

**Pull request merged**